Makefile

```diff
@@ -18,6 +19,20 @@ setup:
 	vagrant up
 	vagrant ssh-config > ssh_config
 
+
+batch:
```

Comment:
Can you add comments to the new targets to describe what they do. And maybe add an example to the README like how to test only the windows group.